.dockerignore

```diff
@@ -2,6 +2,8 @@ _vendor
 storage/engine/engine.pc
 cockroach
 build/deploy
+.git/FETCH_HEAD
+.git/modules
```

Comment:
No, thought about tossing it but didn't. It will be useful once we tag releases because then we will want to be able to do git scripting in the container.